public/app/features/dashboard/dashgrid/DashboardEmpty.tsx

```diff
@@ -107,6 +107,7 @@ function getStyles(theme: GrafanaTheme2) {
       flexDirection: 'column',
       maxWidth: '890px',
       gap: theme.spacing.gridSize * 4,
+      paddingTop: theme.spacing(12),
```

Comment:
This is affecting mobile 

![Captura de pantalla 2023-05-09 a las 11 23 16](https://user-images.githubusercontent.com/5699976/237053575-79b7d5c5-8bdd-425f-a119-9e5246910df0.png)

Reply:
@ivanortegaalba fixed, added a media query